### DISCUSSION TITLE
Hotkey to toggle fullscreen and windowed mode.

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#2966] [Ideas]
alt + enter to f12 seem to be common hotkeys for this

**Comment by ZippyZolton** [comment]
Alt + Enter is somewhat common, but the only program I actively use that in is RPCS3. F11 is the most used fullscreen key in my personal experience.

**Comment by Zedb0T** [comment]
Opened a PR [here](https://github.com/open-goal/jak-project/pull/3230/files) and defaulted the key to F12 - Open for suggestions/Improvements/discussion whatever. Can also set your own key in debug-settings.json. 

(Maybe some magic to make this mappable in game is worth it in the future)
